src/components/simple-cta/simple-cta.scss

```diff
@@ -37,7 +37,8 @@ a:hover {
     text-transform: none;
     font-weight: normal;
     display: flex;
-    justify-content: space-between;
+    justify-content: start;
```

Comment:
@Fupete we could just omit this declaration, as it works the same way without specifying it 👍